src/index.js

```diff
@@ -123,26 +123,119 @@ export default class ToggleBlock {
+    input.addEventListener('focus', this.setDefaultContent.bind(this));
+    input.addEventListener('focusout', this.setDefaultContent.bind(this));
+
+    defaultContent.classList.add('toggle-block__content-default');
+    defaultContent.setAttribute('hidden', true);
+    defaultContent.innerHTML = 'Empty toggle. Click or drop blocks inside.';
+    defaultContent.addEventListener('click', this.clickInDefaultContent.bind(this));
+
     this.wrapper.appendChild(icon);
     this.wrapper.appendChild(input);
+    this.wrapper.appendChild(defaultContent);
+  }
+
+  /**
+   * Adds the actions to do when the default content is clicked.
+   */
+  clickInDefaultContent() {
+    const originalIndex = this.api.blocks.getCurrentBlockIndex();
+    const foreignKey = this.wrapper.id;
+    const index = originalIndex + 1;
+    const id = crypto.randomUUID();
+
+    this.api.blocks.insert();
+
+    const newBlock = this.api.blocks.getBlockByIndex(index);
+
+    newBlock.holder.firstChild.firstChild.classList.add('toggle-block__item');
+    newBlock.holder.setAttribute('foreignKey', foreignKey);
+    newBlock.holder.setAttribute('id', id);
+
+    document.getElementById(id).firstChild.firstChild.focus();
+    this.setDefaultContent();
+  }
+
+  /**
+   * Sets the default content. If the toggle has no other blocks inside it,
+   * so remove the hidden tag in the default content, otherwise adds the hidden tag.
+   */
+  setDefaultContent() {
+    const children = document.querySelectorAll(`div[foreignKey="${this.wrapper.id}"]`);
+
+    if (children.length === 0) {
+      this.wrapper.lastChild.removeAttribute('hidden');
+    } else {
+      this.wrapper.lastChild.setAttribute('hidden', true);
+    }
+  }
+
+  /**
+   * If the toggle root is empty and the key event received is 'backspace'
+   * the toggle root is removed.
+   *
+   * @param {KeyboardEvent} e - key down event
+   */
+  removeToggle(e) {
+    if (e.code === 'Backspace' && this.wrapper.children[1].textContent.length === 0) {
+      const index = this.api.blocks.getCurrentBlockIndex();
+      this.api.blocks.delete(index);
+      this.api.blocks.insert();
+    }
+  }
+
+  /**
+   * If the toggle root is empty and the key event received is 'backspace'
+   * or 'enter', its content is cleared so that the visible placeholder
+   * is set through the css.
+   *
+   * @param {KeyboardEvent} e - key up event
+   */
+  setPlaceHolder(e) {
+    if (this.wrapper.children[1].textContent.length === 0) {
+      if (e.code === 'Backspace' || e.code === 'Enter') {
+        this.wrapper.children[1].textContent = '';
+      }
+    }
+  }
+
+  /**
+   * Establishes the icon color, if the toggle has no other blocks inside it,
+   * sets the gray color, otherwise sets the black color.
+   */
+  calculateChildren() {
+    const children = document.querySelectorAll(`div[foreignKey="${this.wrapper.id}"]`);
+    this.wrapper.firstChild.style.color = children.length === 0 ? 'gray' : 'black';
```

Comment:
@juliansg022 please use object destructing.

Reply:
@juanhurtado10 You're right, I'll check again the code to avoid using too many function calls here and in other places.